### ISSUE TITLE
ParentCallSpacing tries to format "parent" return typehint

### ISSUE THREAD

**Issue by janlanger** [slevomat/coding-standard#874]
Example code:

```php
class Foo
{

	public function bar(): self
	{
		return new self();
	}

}

class Baz extends Foo
{

	public function bar(): parent
	{
		return new Foo();
	}

}
```

Running phpcs with ParentCallSpacing sniff enabled reports error on line `public function bar(): parent` - `Expected 0 lines before "parent", found -1.`

Sniff should probably not check spacing around parent return type hint...

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/16fe6597c881c9ceb6ef54bbb384d42aab9a3188
